sql/sql_test.go

```diff
@@ -113,7 +113,6 @@ func TestMain(m *testing.M) {
 // log doesn't work.
 func assertNoErr(e error) {
 	if e != nil {
-		fmt.Println(e)
```

Comment:
I will revert this change due to the comment "log doesn't work" by @wangkuiyi 

https://github.com/sql-machine-learning/sqlflow/blob/572b23a37ccfd9f4041ede14508c5c76aab62183/sql/db_test.go#L45-L47